client/lib/checkout/index.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Internal dependencies
+ */
+import { cartItems } from 'lib/cart-values';
+import { managePurchase } from 'me/purchases/paths';
+
+export function getExitCheckoutUrl( cart, siteSlug ) {
+	let url = '/plans/';
+
+	if ( cartItems.hasDomainRegistration( cart ) ) {
+		url = '/domains/add/';
```

Comment:
We will have one for plans [very soon](https://github.com/Automattic/wp-calypso/commit/b482cd1e837928fac90b0f36876f8f82e11df187).